main.go

```diff
@@ -247,3 +273,17 @@ func newKubeCmd(c *cli.Context, args []string) *exec.Cmd {
 
 	return exec.Command(kube, args...)
 }
+
+// listDirectory returns a recursive list of all file under an directory, or an error
+func listDirectory(path string) ([]string, error) {
+	var list []string
+	// QUESTION: should we filter files i.e. only json or yaml??
```

Comment:
JSON is not supported yet and I guess not many people would use json anyway. What we should probably do is to try and load each file to check what kind they are and maybe sort depending on their kind?